### PR TITLE
Added the missing CR from Adrian as PullRequest to next

### PR DESCRIPTION
**Pull request by ue71603**
Added the missing Accessibility elements we think we should use for modeling the accessibility fully.

### CODE REVIEW

**Comment by Aurige** [comment]
We may also consider "TactileWarningStrip" which is more and more requested by associations here in France

File: xsd/netex_framework/netex_genericFramework/netex_accessibility/netex_acsb_limitations.xsd [hunk 1]
```diff
@@ -66,6 +67,8 @@ Rail transport, Roads and Road transport
 			<xsd:element ref="StepFreeAccess" minOccurs="0"/>
 			<xsd:element ref="EscalatorFreeAccess" minOccurs="0"/>
 			<xsd:element ref="LiftFreeAccess" minOccurs="0"/>
+			<xsd:element ref="RampFreeAccess" minOccurs="0"/>
+			<xsd:element ref="GuideDogAccess" minOccurs="0"/>
```
Comment (ue71603) [inline xsd/netex_framework/netex_genericFramework/netex_accessibility/netex_acsb_limitations.xsd:71]: already existing in EIAP branch. but  in SensoryLimitationGroup

File: xsd/netex_framework/netex_genericFramework/netex_accessibility/netex_acsb_limitations.xsd [hunk 2]
```diff
@@ -105,6 +108,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Whether a PLACE has audible signals for the visually impaired.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element ref="TactileGuidingStripsAvailable" minOccurs="0">
```
Comment (ue71603) [inline xsd/netex_framework/netex_genericFramework/netex_accessibility/netex_acsb_limitations.xsd:111]: Ordered a bit differently and the name is TactileGuidanceAvailable

File: xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd [hunk 3]
```diff
@@ -95,6 +96,13 @@ Rail transport, Roads and Road transport
 							<xsd:group ref="ZoneGroup"/>
 						</xsd:sequence>
 						<xsd:group ref="PlaceGroup"/>
+						<xsd:element ref="AccessibilityAssessment" minOccurs="0"/>
```
[the remark this answers is not in the document]
Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd:99]: I think we did that already in a different setting.

File: xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd [hunk 4]
```diff
@@ -115,6 +123,13 @@ Rail transport, Roads and Road transport
 		<xsd:complexContent>
 			<xsd:extension base="Place_VersionStructure">
 				<xsd:sequence>
+					<xsd:element ref="AccessibilityAssessment" minOccurs="0"/>
```
Comment (ue71603) [inline xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd:126]: same here

File: xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd (hunk 3 above)
[the remark this answers is not in the document]
Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd:99]: https://github.com/NeTEx-CEN/NeTEx/pull/275

**Comment by ue71603** [comment]
I think there is nothing left in this PR that was not done elsewhere. 
I think we can close it.
@skinkie  @Aurige  do you concur?

**Comment by Aurige** [comment]
What is in this PR is Ok for me (provided that it goes to 'next' so it can be discussed/validated by the group during the incoming revision)... and if it has been split in different PR's that's OK (which one are now managing RampFreeAccess and GuidedDogAccess ? and TactileWarningStrip and TactileGuidingStrip ?)

**Comment by Aurige** [comment]
Ok... I would favour having both TactileGuidanceStrip and TactileWarningStrip (it's 2 different features, and it's weird to mention one without the other)

**Comment by ue71603** [comment]
Everything done in other PR.

**Comment by trurlurl** [comment]
Removing "needs documentation update" label since closed without having been merged.

**Pull request closed without merging**